### PR TITLE
Feature/fix typos in readme

### LINKED ISSUE #6: Can you clarify `report_format` setting please?
## Background
The description of `report_format` is somewhat confusing.

## Question
The `report_format` setting offers three options 
*  console
* csv
* json

While the later two are indeed formats, the first is an output channel how I understand it. So what exactly this means?
Is first a simple text output to console? And the other two file based output in given format?
Or what is the logic of the setting, please?

### PR DESCRIPTION
**Pull request by miroslavpojer**
- Fixed typo issue in README: `_` instead of `-`.
- Update the code to where it worked with `_` too.
- Updated unit tests.

Also closes #6

### CODE REVIEW

**Comment by miroslavpojer** [comment]
Release notes:
- Fixed usage of '_' and '-' in code and documentation.
- Refactoring of action input logic.

**Review by Zejnilovic — APPROVED** [review]

File: src/filename_inspector.py [hunk 1]
```diff
@@ -69,28 +76,32 @@ def run():
         violations = find_non_matching_files(name_patterns, paths, excludes)
         violation_count = len(violations)
 
-        set_output('violation_count', str(violation_count))
+        set_action_output('violation-count', str(violation_count))
+
+        if verbose_logging:
+            print(f'Total violations: {violation_count}')
```
Comment (benedeki) [inline src/filename_inspector.py:82]: What's the difference of `print` and `logging`?

Comment (miroslavpojer) [reply src/filename_inspector.py:82]: Print output is independent of logging settings.
What level would you propose?

Comment (benedeki) [reply src/filename_inspector.py:82]: That's extra weird. There are good chances, that verbose set to `true` will print out the same info twice.
My suggestion:
* treat `console` as _console only_ output, but print there even if other option is used (and adjust descripton accordingly)
* if `verbose` is _false_, print only the violations number, if _true_ print all violations

Or use other logic. But this is IMHO wrong.

Comment (miroslavpojer) [reply src/filename_inspector.py]: Improved.

**Review by benedeki — APPROVED** [review]

**Pull request merged**